include/vecops.hxx

```diff
@@ -81,7 +81,7 @@ const Field2D Div(const Vector2D& v, CELL_LOC outloc = CELL_DEFAULT,
 const Field3D Div(const Vector3D& v, CELL_LOC outloc = CELL_DEFAULT,
                   const std::string& method = "DEFAULT");
 
-const Field2D Div(const Vector2D &v, const Field2D &f);
+const Field2D Div(const Vector2D& v, const Field2D& f, CELL_LOC outloc = CELL_DEFAULT);
```

Comment:
Think this should have a `const std::string& method = "DEFAULT"` argument too
https://github.com/boutproject/BOUT-dev/blob/eeed651ac6ce00694d6debfe3b28ed00245bade0/src/field/vecops.cxx#L182-L183

Reply:
Thanks, fixed!